mks/service/mks_client_02_04.py

```diff
@@ -62,7 +62,7 @@ def extract(xml_data):
         tree = BeautifulSoup(xml_data, features='lxml-xml')
         if tree.find('Body') is None:
             logging.error("No Body tag. no data for person")
-            raise NoResultException
+            raise NoResultException()
```

Comment:
Niet per sé nodig, maar kan me voorstellen dat dit meer in lijn is met de rest van het project.

Reply:
Wist niet eens dat het niet nodig was, goed om te weten!
idd, dit is zoals het overal gebeurd.